configs/data-driven-forms.json

```diff
@@ -12,7 +12,7 @@
     "lvl3": ".DocSearch-content h4",
     "lvl4": ".DocSearch-content h5",
     "lvl5": ".DocSearch-content h6",
-    "text": ".DocSearch-content p, .DocSearch-content li"
+    "text": ".DocSearch-content p, .DocSearch-content li, .DocSearch-content pre, .DocSearch-content .DocTable, .DocSearch-content .DocCode, .DocSearch-content .DocRawComponent"
```

Comment:
These selectors target code element or element that are too big. We do not recommend to index these kind of elements https://community.algolia.com/docsearch/faq.html#can-you-index-code-samples

Reply:
Fixed!

Reply:
Great thanks